Source/CNTK/CNTK.cpp

```diff
@@ -690,7 +691,7 @@ int wmainOldCNTKConfig(int argc, wchar_t* argv[])
 
 #ifndef CPUONLY
     ConfigValue val = config("deviceId", "auto");
-    if (!EqualCI(val, "cpu") && !EqualCI(val, "auto"))
+    if (!EqualCI(val, "cpu") && !EqualCI(val, "auto") && !EqualCI(val, "localRank"))
```

Comment:
How does this work with multi-tenant cases? Say, we have a 8-GPU server running 2 4-GPU jobs. Each job would have local rank 0-3, so GPU 4-7 is not used?